CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 # UNRELEASED
 
+### feat: Default to LMDB backend on macOS
+
+MacOS builds of dfx now use the LMDB artifact pool backend by default. If you encounter any
+issues with the new backend, you can switch to the old one using the `--rocksdb` flag on 
+`dfx start`.
```

Comment:
Please create a discussion on https://github.com/dfinity/sdk/discussions and link to it here (if developers are encountering problems, we want to hear about it)